mycroft/tts/__init__.py

```diff
@@ -333,14 +333,15 @@ class TTSFactory(object):
     from mycroft.tts.mary_tts import MaryTTS
     from mycroft.tts.mimic_tts import Mimic
     from mycroft.tts.spdsay_tts import SpdSay
-
+    from mycroft.tts.ibm_tts import WatsonTTS
```

Comment:
The means WatsonTTS should be added to the requirements.txt, correct?

Reply:
hum no, WatsonTTS is the engine class i made, only requirements is requests, i used the standard remoteTTS class available